inventory/redhat.cf

```diff
@@ -12,11 +12,13 @@ bundle common inventory_redhat
       comment => "derived from Red Hat",
       meta => { "inventory", "attribute_name=none" };
 
+      "inventory_redhat_have_python_symlink" expression => fileexists("/var/cfengine/bin/python");
+
+    inventory_redhat_have_python_symlink::
       "cfe_yum_package_module_supported" -> { "CFE-2602" }
         comment => "Here we see if the version of python found is acceptable for
                     the yum package module. We use this guard to prevent errors
                     related to missing python modules.",
-        expression => returnszero("python -V 2>&1 | grep ^Python | cut -d' ' -f 2 | ( IFS=. read v1 v2 v3 ; [ $v1 -ge 3 ] || [ $v1 -eq 2 -a $v2 -ge 4 ] )",
+        expression => returnszero("/var/cfengine/bin/python -V 2>&1 | grep ^Python | cut -d' ' -f 2 | ( IFS=. read v1 v2 v3 ; [ $v1 -ge 3 ] || [ $v1 -eq 2 -a $v2 -ge 4 ] )",
```

Comment:
Oh i guess we need to keep this crap because we are checking for python greater than 2.4

Reply:
Yes.